src/test/java/api/post/CreateANewBoard.java

```diff
@@ -26,10 +29,28 @@ public void createBoardTest() {
         Assert.assertNotNull(response, "Board creation failed");
         Assert.assertNotNull(response.getId(), "Board ID is null");
         Assert.assertEquals(response.getName(), "Test board", "Board name doesn't match");
+
+        deleteBoard();
     }
 
-    @AfterMethod
     public void deleteBoard() {
         boardRestTestClient.deleteBoardIfExist(ID_BOARD);
     }
-}
+
+    @Test(description = "AS2-36")
+    @Description(" Negative: Create a Board with invalid name ")
+    public void createBoardWithEmptyNameTest() {
+
+        boardRestTestClient.tryToCreateABoard(constructDefaultNoNameBoardKeyValue(" "), HTTP_BAD_REQUEST);
+
+        String jsonResponse = "{ \"message\": \"invalid value for name\", \"error\": \"ERROR\" }";
+        JsonPath jp = new JsonPath(jsonResponse);
+        String message = jp.getString("message");
+
+        assertThat(message).isEqualTo("invalid value for name");
+    }
+
+    private Map<String, String> constructDefaultNoNameBoardKeyValue(String boardName) {
+        return Map.of("name", boardName);
+    }
```

Comment:
Try use:

Map<String, String> nonNameKeyValue= new HashMap<>();
        nonNameKeyValue.put("name", " "); 
        
        This "construct" is pointless